### PR TITLE
vocab review

### PR DESCRIPTION
**Pull request by elf-pavlik**
the first commit for discussing details inline

### CODE REVIEW

File: proposals/specification/interop.ttl [hunk 1]
```diff
@@ -230,6 +226,7 @@ rdfs:label "Terms for application interoperability in Solid"@en .
   a rdf:Property ;
   rdfs:comment "Links to instances of the shape tree in a data registration"@en ;
   rdfs:isDefinedBy <> ;
+  rdfs:domain :DataRegistry ;
```
Comment (justinwb) [inline proposals/specification/interop.ttl:229]: We've not been using this in the DataRegistry of late (relying on ldp:contains instead). We do use this in Access Needs, Data Consents, and Data Grants though.

File: proposals/specification/interop.ttl [hunk 2]
```diff
@@ -486,23 +489,9 @@ rdfs:label "Terms for application interoperability in Solid"@en .
   rdfs:comment "Agent or mode the Application authenticates as"@en ;
   rdfs:isDefinedBy <> ;
   rdfs:domain :AccessNeedGroup ;
+  rdfs:range [ owl:oneOf ( :SocialAgent :Application ) ] ;
```
Comment (elf-pavlik) [inline proposals/specification/interop.ttl:492]:
```suggestion
  rdfs:range [ owl:unionOf ( :SocialAgent :Application ) ] ;
```

File: proposals/specification/interop.ttl [hunk 3]
```diff
@@ -335,32 +346,44 @@ rdfs:label "Terms for application interoperability in Solid"@en .
```
Comment (elf-pavlik) [inline proposals/specification/interop.ttl:346]:
```suggestion

:delegationOfGrant
  rdfs:comment "Data Grant which is being delegated"@en ;
  rdfs:isDefinedBy <> ;
  rdfs:domain :DelegatedDataGrant ;
  rdfs:range :DataGrant ;
  rdfs:label "Delegation of Grant"@en .
```

**Review by justinwb — APPROVED** [review]
Looks good!

**Pull request merged**